extensions/wikia/PortableInfobox/templates/PortableInfoboxItemData.mustache

```diff
@@ -1,3 +1,7 @@
+{{!
+	This template is linked with of mercury/front/templates/main/components/infobox-builder-item-data.hbs
+	Changing this template requires modifying the other in mercury repo.
+}}
 <div class="pi-item pi-data pi-item-spacing pi-border-color">
 	{{#label}}
 		<h3 class="pi-data-label pi-secondary-font">{{{label}}}</h3>
```

Comment:
This templates are used only in portable infoboxes, which is build by parser. Here we only inform that this code was duplicated in mercury.